### PR TITLE
Upgrade mocha package

### PR DESCRIPTION
**Pull request by volodymyrmakarchuk-okta**
## Description:
- **What's changed?** 
- **Is this PR related to a Monolith release?** 

### Resolves:

* [OKTA-334549](https://oktainc.atlassian.net/browse/OKTA-334549)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
     "cypress": "^7.6.0",
     "if-env": "^1.0.4",
     "jest": "^24.9.0",
-    "mocha": "^5.2.0",
+    "mocha": "^9.1.0",
```
Comment (shawyu-okta) [inline package.json:42]: No breaking changes that affect us?

**Review by shawyu-okta — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (volodymyrmakarchuk-okta) [reply package.json:42]: tests run properly, I don't see any other area where this package could have an impact

File: yarn.lock [hunk 2]
```diff
@@ -7780,11 +7851,6 @@ minimist-options@^3.0.1:
     arrify "^1.0.1"
     is-plain-obj "^1.1.0"
 
-minimist@0.0.8:
-  version "0.0.8"
-  resolved "https://registry.yarnpkg.com/minimist/-/minimist-0.0.8.tgz#857fcabfc3397d2625b8228262e86aa7a011b05d"
-  integrity sha1-hX/Kv8M5fSYluCKCYuhqp6ARsF0=
```
Comment (shawyu-okta) [inline yarn.lock:7786]: 👍  This is the package+version we're trying to eliminate

File: package.json [hunk 3]
```diff
@@ -39,7 +39,7 @@
     "cypress": "^7.6.0",
     "if-env": "^1.0.4",
     "jest": "^24.9.0",
-    "mocha": "^5.2.0",
```
Comment (robertnicolo-okta) [inline package.json:42]: I pulled down this branch and ran "yarn install".  It upgraded mocha to 9.1.0 but kept cypress at 7.6.0.  I ran through a basic functionality test of the site and it looked fine. (Also  cypress tests ran fine.) Should cypress remain at 7.6.0 or should we upgrade to 8.3.0 as well?

**Review by robertnicolo-okta — APPROVED** [review]
I approve but have a question about the version of cypress.  See my comment.

File: package.json (hunk 3 above)
Comment (volodymyrmakarchuk-okta) [reply package.json:42]: the main goal of this PR was to upgrade `minimist` package that is used by mocha, to do so we upgrade mocha that automatically has upgraded `minimist`. Cypress upgrade was not a part of this task

**Pull request merged**